### DISCUSSION TITLE
Dependent libraries of rabbitmq

### DISCUSSION THREAD

**Discussion by Vishnu-0809** [rabbitmq/rabbitmq-server#11017] [Questions]
How can I know what are the dependent libraries/ third party packages that are required for compiling rabbitmq source code.
Not able to find it from the readme or any other documentation.

**Comment by lukebakken** [comment]
You need Erlang and GNU `make`, pretty much.

Please point out where our documentation is insufficient - https://www.rabbitmq.com/docs/build-server

**Reply by Vishnu-0809** [reply]
Thanks for the reply, maybe I have to rephrase my question:
When we are building the rabbitmq source code for sure we require erlang libraries or any other language packages which is required for the source code compilation
So my primary goal is to understand what are the libraries that are used for this build process and where are they picked from (which repo basically)

**Reply by lukebakken** [reply]
We don't have time to go into this much detail. You should build RabbitMQ yourself and learn on the way. If you have trouble with the build, you can ask for assistance in these places, but we do not prioritize support for OSS users -
* https://github.com/rabbitmq/rabbitmq-server/discussions
* https://groups.google.com/g/rabbitmq-users
